poptypes/aristocrats.txt

```diff
@@ -811,10 +811,6 @@ promote_to = {
 		modifier = {
 			factor = -100
 			soldiers = 0.1
-			country = {
-				has_global_flag = unciv_10
-				civilized = no
-			}
 		}
```

Comment:
Remove this check entirely. There already is a -100 factor when soldiers hit 10%